src/components/Filter/index.vue

```diff
@@ -70,8 +89,22 @@ export default {
   data() {
     return {
       name: '',
+      types: [],
+      filterTypes: [],
     };
   },
+  mounted() {
+    axios
+      .get('https://pokeapi.co/api/v2/type')
```

Comment:
hello @sheilaazhar it's a good practice to store any credentials (including API Endpoint) on `.env` file. The benefit of using Environment Variables are:

1. We can hide the API from the source code
2. We can easily change the API Endpoint on the deployment configuration
3. We create a habit to securely store credentials

My suggestion is to create a `.env` file and add them on `.gitignore`

```bash
# .env
VUE_APP_API_ENDPOINT="https://pokeapi.co/api/v2"
```

we can access the ENV Variable using `process.env`

```js
axios
      .get(`${process.env.VUE_APP_API_ENDPOINT}/type`)
```

ref: https://cli.vuejs.org/guide/mode-and-env.html#environment-variables

Reply:
Oh I see, thank you @Ibwedagama I have fixed [it](https://github.com/sheilaazhar/fe-onboarding-mini-task/pull/4/commits/6f9ba7d8b41018f20ec0c04488fb81df6ed01529)